README.md

```diff
@@ -346,6 +346,21 @@ To stop sharing existing factors created with **App Groups**, uncheck the **App
 
 > This will restrict access to the factors and will not affect the main application in which the data was initially created.
 
+---
+
+## Allow Factors migration
+
+By default, all factors are stored using the accessibility attribute [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible) with a value of [kSecAttrAccessibleAlwaysThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessiblealwaysthisdeviceonly). This means that the item stored with this value can be restored to the same device that created a backup, but it isn't migrated when restoring another device's backup data.
+
+To allow [Data migration from iPhone to iPhone](https://support.apple.com/en-us/HT210216) or [Backup restoration](https://support.apple.com/en-us/HT204184) use the following `setAttrAccessible(.afterFirstUnlock)` method to change the accessibility attribute [kSecAttrAccessible](https://developer.apple.com/documentation/security/ksecattraccessible) with a value of [kSecAttrAccessibleAlwaysThisDeviceOnly](https://developer.apple.com/documentation/security/ksecattraccessiblealwaysthisdeviceonly) before build the Twilio Verify instance. For example:
```

Comment:
Should the value be: [kSecAttrAccessibleAfterFirstUnlock](https://developer.apple.com/documentation/security/ksecattraccessibleafterfirstunlock) (value and link)?

Reply:
Also, we will need to clarify previous factors will be using the previous value (no migration) but new factors will use the defined value, and the value by default we are using